.travis.yml

```diff
@@ -9,6 +9,8 @@ before_install:
   - wget 'https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.7-linux-x86_64.tar.bz2'
   - tar xf phantomjs-1.9.7-linux-x86_64.tar.bz2
   - export PATH=$PATH:phantomjs-1.9.7-linux-x86_64/bin/
+  - psql -c "CREATE USER accounts WITH SUPERUSER PASSWORD 'accounts'" -U postgres
```

Comment:
@lakshmivyas I changed it to:

```bash
psql -c "CREATE USER accounts WITH CREATEDB PASSWORD 'accounts'" -U postgres
psql -c 'CREATE DATABASE "accounts-testing" WITH OWNER accounts' -U postgres
psql -c 'GRANT ALL PRIVILEGES ON DATABASE "accounts-testing" TO accounts' -U postgres
```

I have to give the user permission to create db otherwise I get something like `PG::InsufficientPrivilege: ERROR:  permission denied to create database` when I run the tests.

Does this look ok?